apps/antalmanac/src/components/Calendar/Toolbar/CustomEventDialog/DaySelector.tsx

```diff
@@ -30,24 +26,29 @@ const DaySelector = ({ days = [false, false, false, false, false, false, false],
             sx={{
                 display: 'flex',
                 flexDirection: 'row',
-                justifyContent: 'center',
-                padding: 5,
+                justifyContent: 'space-between',
+                marginTop: 10,
+                marginBottom: 10,
+                marginLeft: -5,
+                marginRight: -5,
```

Comment:
```suggestion
                marginTop: 10,
                marginBottom: 10,
                marginLeft: -5,
                marginRight: -5,
```
issue: negative margins (although not always bad) are usually a sign of an antipattern -- they introduce more code to "cancel out" other code. Removing this actually makes the component clearer and more maintanable

suggestion: when adding negative margin, ask yourself where the source of the unwanted spacing is coming from